cli/asc.js

```diff
@@ -278,7 +278,7 @@ exports.main = function main(argv, options, callback) {
   while (asconfig) {
     // merge target first, then base options, then extended asconfigs
     if (asconfig.targets && asconfig.targets[target]) {
-      optionsUtil.merge(exports.options, args, asconfig.targets[target])
+      optionsUtil.merge(exports.options, args, asconfig.targets[target]);
```

Comment:
Note that merge returns the new set of options, doesn't modify given sets

Reply:
Oh right! Yeah. we need to set `args` each time. Right?